compiler/rustc_ast/src/ast.rs

```diff
@@ -1709,20 +1709,19 @@ pub struct MetaItemLit {
     pub span: Span,
 }
 
-/// Same as `Lit`, but restricted to string literals.
+/// A string literal representing an ABI value.
```

Comment:
I suspect that `StrLit` will be used in other places too when https://github.com/rust-lang/rust/issues/95625 is fixed, so I'd keep it as is.

Reply:
Ok, I have replaced that commit with one that only adjust the comments on `StrLit`.